ietf-layer0-types.yang

```diff
@@ -504,6 +576,18 @@ module ietf-layer0-types {
       "all elements must use power (dBm)";
   }
 
+  identity operational-mode {
```

Comment:
Based on this change, one or two examples shall be put into an Appendix of draft-ietf-ccamp-optical-impairment-topology-yang explaining how vendors can define their operational modes.